Updated version # button and rest of footer to color match site

djjones95 opened the pull request:
Got the version number button to be the same color as the privacy button is was next to. Also got the copyright BucStop to be the same color, it was gray before.

djjones95 committed:
Updated version # button and rest of footer to color match site

Got the version number button to be the same color as the privacy button is was next to. Also got the copyright BucStop to be the same color, it was gray before.

Dylemperor reviewed (approved):
Looks good. Color looks like the right one

djjones95 committed:
removed div on site.css to make copyright and BucStop colors revert back to gray

did this to remove any confusion between the links that are clickable next to it. Makes the links stick out and look "clickable" compared to the copyright BucStop which is not clickable.

Heroshane1 commented:
Looks nice to me.

Heroshane1 reviewed (approved):
Looks good to me.
